doc/source/index.rst

```diff
@@ -0,0 +1,333 @@
+.. pytroll-collectors documentation master file, created by
+   sphinx-quickstart on Tue Dec 22 16:13:24 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to pytroll-collectors's documentation!
+==============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+Pytroll-collectors is a collection of scripts and modules to aid in
+operational processing of satellite reception data.
+The scripts run continuously (like daemons) and
+communicate with each other using `posttroll`_ messages.
+
+For example, a chain for processing MetOp AVHRR data from direct
```

Comment:
"Metop" it is! Use the Oscar naming and the one (mostly) used at EUMETSAT. ESA is guilty of MetOp

Reply:
Fixed.